routes/exchange.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express')
+const router = express.Router()
+const exchangeController = require('../controllers/exchangeController')
+const authenticate = require('../middlewares/authenticate')
+const { postCurrencyRates, patchExchange } = require('../middlewares/validators/exchange')
+const { authorizeUser } = require('../middlewares/authorization')
+
+/**
+ * @swagger
+ * /exchange/rates:
+ *   get:
+ *     summary: Get exchange rate
+ *     tags:
+ *       - exchange
+ *     responses:
+ *       200:
+ *         description: Return exchange
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/exchange'
+ *       500:
+ *         description: badImplementation
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.get('/rates', exchangeController.getExchangeRate)
+
+/**
+ * @swagger
+ * /exchange/rates:
+ *   get:
+ *     summary: post exchange rate
+ *     tags:
+ *       - exchange
+ *     responses:
+ *       201:
+ *         description: Return exchange
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/exchange'
+ *       401:
+ *         description: Unauthorized
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/unAuthorized'
+ *       500:
+ *         description: badImplementation
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.post('/rates', authenticate, authorizeUser('superUser'), postCurrencyRates, exchangeController.createExchangeRate)
```

Comment:
`superUser` can come from constants?
Implemented in this PR: https://github.com/Real-Dev-Squad/website-backend/pull/357
You can copy paste maybe?